### DISCUSSION TITLE
How to enable Dark Mode?

### DISCUSSION THREAD

**Discussion by Kalagaar** [danny-avila/LibreChat#1301] [Q&A] [answered]
### Contact Details

florestano.pepe@gmail.com

### What is your question?

It seems that i cannot enable Dark Mode.
No option present (better explaination: no option button at all)
Surely it's something that i don't understand...

### More Details

I use OpenAI API Key directly from .env, as also an external (local) MongoDB. 
Installation done buliding the container from the git clone

### Screenshots

<img width="1718" alt="LibreChat" src="https://github.com/danny-avila/LibreChat/assets/5342808/8b188fe7-f9b1-4cd8-8a99-d86d2cbc9eed">

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
There is a small grayish arrow on the left hand side most side of the screen. This will open the nav where you can change the settings by clicking your username

**Comment by Kalagaar** [comment]
Thanks. I was becoming crazy about it. Now i feel stupid and a little blind 😅

**Reply by danny-avila** [reply]
No worries and thank you for asking!

**Comment by ilyesjb** [comment]
Enable Dark Mode by Default in LibreChat

Open the following file:

C:\LibreChat\node_modules\@librechat\frontend\src\App.jsx


Find this code block:
`<ThemeProvider
          {...(envTheme && { initialTheme: system, themeRGB: envTheme })}
        > `

 

Modify the ThemeProvider line to force the dark theme:
 `<ThemeProvider  {...(envTheme && { initialTheme: 'dark', themeRGB: envTheme })}>`


LibreChat will start in dark mode by default, even if no theme is saved in localStorage or defined via environment variables.
